artifacts/definitions/Windows/Detection/TemplateInjection.yaml

```diff
@@ -0,0 +1,108 @@
+name: Windows.Detection.TemplateInjection
+description: |
+    This content will detect injected templates in Office documents.   
+    
+    Template injection is a form of defence evasion where a malicious macro 
+    is loaded into an OOXML document via a resource file masquerading as 
+    an office template.   
+
+author: Matt Green - @mgreen27
+
+reference:
+  - https://attack.mitre.org/techniques/T1221/
+  - https://www.sans.org/reading-room/whitepapers/testing/template-injection-attacks-bypassing-security-controls-living-land-38780
+
+parameters:
+  - name: SearchGlob
+    description: Glob to search
+    default: C:\Users\**\*.{docx,dotx,xlsx,xltx,pptx,potx,ppsx,sldx}
+  - name: TemplateRegex
+    description: Regex to search inside resource section.
+    default: '^(https?|smb|\\\\\\\\)'
+  - name: UploadDocument
+    type: bool
+    description: Select to upload document on detection.
+  
+sources:
+  - precondition: SELECT OS From info() where OS = 'windows'
+  
+    queries: 
+     - |
+      -- Find target docs
+      LET office_docs = SELECT  *
+            FullPath AS OfficePath,
+            timestamp(epoch=Mtime.Sec) as OfficeMtime,
+            Size as OfficeSize,
+            url(
+                scheme="file", path=FullPath, fragment="/**").String
+        FROM glob(globs=SearchGlob)
+
+      -- select zip members inside the doc that have some content.
+      LET document_parts = SELECT 
+            OfficePath,
+            FullPath AS ZipMemberPath
+        FROM glob(globs=url(
+            scheme="file", path=OfficePath, fragment="/**").String,
+            accessor='zip')
+        WHERE not IsDir and Size > 0 and ZipMemberPath =~ '\\.rels$'
```

Comment:
Probably better to use Mode.IsRegular  https://golang.org/pkg/os/#FileMode.IsRegular   since it covers off on some weird things like links etc.